src/shared/tpm2-util.c

```diff
@@ -133,6 +133,50 @@ static inline void Esys_Freep(void *p) {
                 sym_Esys_Free(*(void**) p);
 }
 
+static int tpm2_get_capability(
```

Comment:
hmm, gcc should prohibit building this, as adding static functions with user is not allowed...

Reply:
what i am trying to say: please make sure your commits are individually compilable. i.e. maybe run:

```
git rebase origin/main -x 'ninja -C build' 
```

before you push something, so that every commit in your push definitely can be compiled.

This is important to use, to make sure we things remain bisectable (at least superficially)

Reply:
ack, i merged commits together so the new function is used in the same commit.